app/components/mini_post/index.ts

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+export {default} from './post/post';
```

Comment:
Does it make sense to move `./post/post` here and make it the index file? It still feels like this file is not doing anything at all.

Reply:
makes sense yes!

Reply:
done, marking it as resolved since you already approved